AppCenterData/AppCenterData/Internal/Client/MSCosmosDb.m

```diff
@@ -46,7 +46,7 @@
 /**
  * Url character set to skip(utf8 encoding).
  */
-static NSString *const kMSUrlCharactersToEscape = @"!*'();:@&=+$,/?%#[] ";
+static NSString *const kMSUrlCharactersToEscape = @"!*'();:@&=+$,/?%#[]\" ";
```

Comment:
Would that match any of the predefined charsets from NSCharsets for instance https://developer.apple.com/documentation/foundation/nscharacterset/1416804-urlpathallowedcharacterset?language=objc?

Reply:
it does not, these are the chars not included in the these sets which doesn't match the above set:

URLFragmentAllowedCharacterSet  "#%<>[\]^`{|}
URLHostAllowedCharacterSet      "#%/<>?@\^`{|}
URLPasswordAllowedCharacterSet  "#%/:<>?@[\]^`{|}
URLPathAllowedCharacterSet      "#%;<>?[\]^`{|}
URLQueryAllowedCharacterSet     "#%<>[\]^`{|}
URLUserAllowedCharacterSet      "#%/:<>?@[\]^`